adops/gam-native.md

```diff
@@ -1,89 +1,92 @@
-<li>Targeting variables aren't sent from Prebid.js to the AdServer. Rather, they're queried at display time.
-<li>A different rendering library is used.
-</ol>
-{:/}
-{% endcapture %}
-{% include alerts/alert_important.html content=version2 %}
-
 * TOC
 {:toc}
 
-{: .alert.alert-info :}
-Manually configuring GAM for Prebid can be a fair amount of work.
-Consider using our official command line tool, [Prebid Line Item Manager](/tools/line-item-manager.html), to create the setup. Using this tool may save you time and help you avoid mistakes.
+This page walks you through the steps required to create a native ad template and a native creative to attach to your Prebid line items in Google Ad Manager (GAM).
+
+{: .alert.alert-success :}
+For complete instructions on setting up Prebid line items in Google Ad Manager, see [Google Ad Manager with Prebid Step by Step](/adops/step-by-step.html).
 
+For more information about Google Ad Manager native ad setup, see the [Google Ad Manager native ads](https://support.google.com/admanager/answer/6366845) documentation.
 
-## 1. Create a native ad
 
-From Google Ad Manager, select **Delivery > Native**. Click **Create Native Ad**.
+## Create a Native Ad
 
-![native delivery](/assets/images/ad-ops/dfp-native/create_prebid_native.png){: .pb-md-img :}
+1. In GAM, select **Delivery** > **Native**.
+2. Click **New native ad** and select **Single ad**.
 
-Select the **HTML & CSS editor** option.
+{: .alert.alert-info :}
+For information on the Multiplex ad option, see the [Traffic Multiplex ads](https://support.google.com/admanager/answer/9428537) GAM documentation.
 
-![HTML editor option](/assets/images/ad-ops/dfp-native/prebid_native_html_option.png){: .pb-md-img :}
+{:start="3"}
+3. Under **HTML & CSS editor**, click **Select**. This will slide out the **New native style** window.
 
-## 2. Define settings for native format
+### Define Settings for Native Format
 
-For **Ad size** you can select a specific size for the ad unit or specify the "fluid" size.  In this case we'll go with **Fluid**.
+1. Under **Ad size** you can select a specific size for the ad unit or specify the "fluid" size.  In this case we'll go with **Fluid**.
+2. Under **Custom format**, select **New format**. (If you’ve already created an ad unit with the format you want, you can select **Existing format** and select the format to apply to this ad unit.)
+3. Enter a **Format name**, such as `pb-native-fluid`.
+4. Click **Add variable**. This will slide out the **New variable** window.
 
-Under **Custom format**, select **New format**. (If you’ve already created an ad unit with the format you want, you can select Existing format and select the format to apply to this ad unit.)
+Every format needs at least one variable. Don't worry, you can add more later. Or not. Either way, GAM requires at least one variable in order to move on to the next step.
```

Comment:
While I appreciate the attempt to lighten this lovely process, I would suggest dropping the "Or not." It could be misinterpreted.